RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py

```diff
@@ -4,122 +4,92 @@
+                                           'keep QIE10DataFrameHcalDataFrameContainer_hcalDigis_ZDC_*',
+                                           'keep ZDCRecHitsSorted_zdcreco_*_*')
+)
+RecoLocalCaloRECO.outputCommands.extend(RecoLocalCaloAOD.outputCommands)
+phase2_hgcal.toModify( RecoLocalCaloRECO, 
+    outputCommands = RecoLocalCaloRECO.outputCommands + ['keep *_HGCalRecHit_*_*',
+                                                         'keep recoCaloClusters_hgcalLayerClusters_*_*',
+                                                         'keep *_hgcalLayerClusters_timeLayerCluster_*',
+                                                         'keep *_hgcalLayerClusters_InitialLayerClustersMask_*',
+                                                         'keep *_hgcalMultiClusters_*_*',
+                                                         'keep *_iterHGCalMultiClusters_*_*'])
+phase2_hfnose.toModify( RecoLocalCaloRECO, 
+    outputCommands = RecoLocalCaloRECO.outputCommands + ['keep recoCaloClusters_hgcalLayerClustersHFNose_*_*',
+                                                         'keep *_hgcalLayerClustersHFNose_timeLayerCluster_*',
+                                                         'keep *_hgcalLayerClustersHFNose_InitialLayerClustersMask_*'])
+ 
+#FEVT content
+RecoLocalCaloFEVT = cms.PSet(
+    outputCommands = cms.untracked.vstring('keep HBHERecHitsSorted_hbheprerecoMB_*_*',
+                                           'keep ZDCDataFramesSorted_*Digis_*_*',
+                                           'keep ZDCRecHitsSorted_*_*_*',
+                                           'keep HcalUnpackerReport_*_*_*')
+)
+RecoLocalCaloFEVT.outputCommands.extend(RecoLocalCaloRECO.outputCommands)
+phase2_hgcal.toModify( RecoLocalCaloFEVT, 
+    outputCommands = RecoLocalCaloFEVT.outputCommands + ['keep *_HGCalRecHit_*_*',
+                                                         'keep *_HGCalUncalibRecHit_*_*',
+                                                         'keep *_hgcalMultiClusters_*_*',
+                                                         'keep *_iterHGCalMultiClusters_*_*',
+                                                         'keep recoCaloClusters_hgcalLayerClusters_*_*',
+                                                         'keep *_hgcalLayerClusters_timeLayerCluster_*',
+                                                         'keep *_hgcalLayerClusters_InitialLayerClustersMask_*'])
+phase2_hfnose.toModify( RecoLocalCaloFEVT,
+    outputCommands = RecoLocalCaloFEVT.outputCommands + ['keep recoCaloClusters_hgcalLayerClustersHFNose_*_*',
+                                                         'keep *_hgcalLayerClustersHFNose_timeLayerCluster_*',
+                                                         'keep *_hgcalLayerClustersHFNose_InitialLayerClustersMask_*'])
 
+RecoLocalCaloAOD.outputCommands.extend(ecalLocalRecoAOD.outputCommands)
```

Comment:
These should go right after the definitions of the "RecoLocalCaloXXX" PSet's, and before the era related customizations